src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -180,31 +180,29 @@ struct EvolutionMetavars {
     static constexpr bool enable_time_dependent_maps = true;
   };
 
-  // Find both horizons in grid frame.
+  // Find horizons in grid frame, but also interpolate
+  // inertial-frame quantities for observers.
   using horizons_vars_to_interpolate_to_target = tmpl::list<
       gr::Tags::SpatialMetric<volume_dim, ::Frame::Grid, DataVector>,
       gr::Tags::InverseSpatialMetric<volume_dim, ::Frame::Grid>,
       gr::Tags::ExtrinsicCurvature<volume_dim, ::Frame::Grid>,
-      gr::Tags::SpatialChristoffelSecondKind<volume_dim, ::Frame::Grid>>;
-  // Observe both horizons in grid frame too.
-  using horizons_tags_to_observe =
-      tmpl::list<StrahlkorperGr::Tags::AreaCompute<::Frame::Grid>,
-                 StrahlkorperGr::Tags::IrreducibleMassCompute<::Frame::Grid>>;
+      gr::Tags::SpatialChristoffelSecondKind<volume_dim, ::Frame::Grid>,
+      gr::Tags::SpatialMetric<volume_dim, ::Frame::Inertial, DataVector>>;
```

Comment:
I added SpatialMetric because the observers depend on it.

Note that the SpatialMetric I added here is in the Inertial frame, whereas the one a few lines above is in the Grid frame.  The AH finder (which runs in the grid frame) needs SpatialMetric in the Grid frame, and the observers (which are computed in the inertial frame) also need SpatialMetric but in the Inertial frame.  So SpatialMetric in both frames are interpolated to the AH.